packages/manager/modules/pci/src/projects/new/components/voucher/controller.js

```diff
@@ -172,15 +173,27 @@ export default class PciProjectNewVoucherCtrl {
   ============================== */
 
   onVoucherFormSubmit() {
+    this.atInternet.trackClick({
+      name: 'PublicCloud_new_project::confirm_voucher',
+      type: 'action',
+    });
     return this.submitVoucher();
   }
 
   onVoucherFormReset() {
+    this.atInternet.trackClick({
+      name: 'PublicCloud_new_project::delete_voucher',
+      type: 'action',
+    });
     return this.resetVoucher();
   }
 
   onAddVoucherBtnClick() {
     this.formVisible = true;
+    this.atInternet.trackClick({
+      name: 'PublicCloud_new_project::add_voucher',
+      type: 'action',
+    });
   }
```

Comment:
**suggestion:** Avoids code duplication
```suggestion
    this.trackClick('add_voucher');
  }
  
  trackClick(label) {
    this.atInternet.trackClick({
      name: `PublicCloud_new_project::${label}`,
      type: 'action',
    });
  }
```

Reply:
To do that, everything exist:
1 - add `trackClick` deps into voucher component -> already implemented here (packages/manager/modules/pci/src/projects/projects.routing.js) 
2 - use it instead of atInternet